tests/lib/cmsis_dsp/svm/src/f16.c

```diff
@@ -41,8 +41,8 @@
 		params[svec_count + (vec_dims * svec_count) + 1]; \
 	const float16_t gamma = \
 		params[svec_count + (vec_dims * svec_count) + 2]
-
-static void test_arm_svm_linear_predict_f16(void)
+#ifdef CONFIG_CMSIS_DSP_FLOAT16
```

Comment:
This should not be necessary.

`f16.c` is only compiled if `CONFIG_CMSIS_DSP_FLOAT16=y` in the first place.

Reply:
> This should not be necessary.
> 
> `f16.c` is only compiled if `CONFIG_CMSIS_DSP_FLOAT16=y` in the first place.

ok,I have changed it,thanks